site/content/user-guide/amqp-adapter.md

```diff
@@ -258,6 +258,39 @@ The table below provides an overview of the properties that must be set on a com
 | *correlation-id* | yes             | *properties*             | *string*     | MUST contain the value of the *correlation-id* property of the command request message. |
 | *status*         | yes             | *application-properties* | *integer*    | MUST contain a status code indicating the outcome of processing the command at the device (see [Command & Control API]({{< ref "/api/Command-And-Control-API.md" >}}) for details). |
 
+### Examples
+
+The AMQP Command-line client can be used to receive and send a command response message from/to a consumer application. The client is used to simulate a device to receive commands and for sending command responses back to the application. The command consumer client is used to simulate an application sending commands to devices and receiving command responses from devices.
+
+Start the AMQP Command-line client, as follows:
+
+    /hono/cli/target$ java -jar hono-cli-*-exec.jar --spring.profiles.active=amqp-adapter-cli --subscribe=true
+    $ 04:13:49.651 [vert.x-eventloop-thread-0] INFO  org.eclipse.hono.cli.AmqpSend - Device is now ready to receive commands
+
+After successfully starting the client, a message indicating that the device is ready to receive commands will be printed to standard output. The device is now waiting to receive commands from applications. To send a command to the device, open a new terminal shell and start the command consumer, as shown below:
```

Comment:
> open a new terminal shell and start the command consumer, as shown below:

command consumer? this seems to be the client for *sending* the command, isn't it?

Reply:
oops, that was a mistake..